client/dashboard/utils/site-storage.ts

```diff
@@ -0,0 +1,17 @@
+import type { SiteMediaStorage } from '../data/types';
+
+const ALERT_FRACTION = 0.8;
+
+export function getStorageAlertLevel( {
+	storage_used_bytes,
+	max_storage_bytes,
+}: SiteMediaStorage ): 'none' | 'low' | 'exceeded' {
```

Comment:
Would it be better to call it `warning` instead of `low`?

Reply:
Yeah, in any case, `low` and `exceeded` seem contradictory. One is about remaining space and one is about used space.

Reply:
I see yeah 😅 what does `none` mean? "no" storage remaining? But here it means no warning.

I'll switching to `'none' | 'warning' | 'exceeded'`

Reply:
Is this done, or are we missing some commits 🤔

Reply:
I don't know what happened to it! 😖 I guess I overwrote it locally? 🤷 
Done now ddc8072e21125a599fd5c74e4f9c962cd9e2626e